### DISCUSSION TITLE
1st Party cookies showing as blocked on the same domain under DomainMismatch in PSAT

### DISCUSSION THREAD

**Discussion by agni-nykaa** [GoogleChromeLabs/ps-analysis-tool#753] [Support Forum] [answered]
**Describe the bug**
While analysing the cookies using PSAT, cookies with the domain same as the request URL's domain visible under the blocked cookies due to DomainMismatch.

**To Reproduce**
1. Launch Chrome and Go to https://www.nykaa.com/estee-lauder-advanced-night-repair-synchronized-multi-recovery-complex/p/833283
2. Open chrome devTools and select Privacy sandbox tab.
4. Filter the cookies by scope as first party. 
6. Click on any of the blocked cookie with same domain as request url and check for the blocked reason.

**Expected behavior**
Cookies shouldn't be shown as blocked cookies with domain same as request url.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: iOS 14.1 
 - Browser: Google Chrome
 - Version: 126.0.6478.127 (Official Build) (arm64)

**Comment by milindmore22** [comment, marked as answer]
Hello @agni-nykaa,

Thank you for reaching out to us.

First-party cookies can be blocked for various reasons unrelated to the deprecation of third-party cookies. One common reason is a `DomainMismatch`. This occurs when the request URL’s domain does not exactly match the cookie’s domain or is not a subdomain of the cookie’s Domain attribute value.

For example, the cookie `EXP_NEW_SIGN_UP` has the domain `www.nykaa.com`, but it was blocked because the requested URL was `nykaa.com`, which did not exactly match.

We hope this explanation helps you better understand the PSAT reports.

I will be converting this issue to support forum discussion, For future reference, please [create support discussion](https://github.com/GoogleChromeLabs/ps-analysis-tool/discussions/new?category=support-forum) if you have any question about the PSAT.
